ultravox/model/ultravox_model_test.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import torch
+from transformers import WhisperConfig
+from transformers.models.whisper import modeling_whisper as whisper
+from ultravox.model import ultravox_model
+
+class TestModifiedWhisperEncoder:
```

Comment:
nit: We don't use classes for our tests. It's probably best to use stand-alone functions for consistency since I don't see the `self` being useful anywhere.